TeamCode/src/main/java/org/firstinspires/ftc/teamcode/AutonomousGrabCube.java

```diff
@@ -45,30 +45,35 @@ public void runOpMode() throws InterruptedException {
         pause();
 
        // crab back
-        drive.lockHeading();
         resetStartTime();
-        while(withinTimeBox(1.33)){
-            drive.autonomousCorrectedDrive(-0.4f,0.0f);
+        while(withinTimeBox(1.5)){
+            drive.Crab(.5f);
             telemetry.update();
         }
         drive.stop();
         pause();
 
        // go to bridge
-        drive.lockHeading();
         resetStartTime();
-        while(withinTimeBox(1.66)){
-            drive.autonomousCorrectedDrive(0.0f,-0.5f);
+        while (withinTimeBox(4)){
```

Comment:
The testing today showed this is very going very far. maybe this should go back to 1.66?